opus/import/populate_obs_instrument_VGPPS_prof.py

```diff
@@ -0,0 +1,581 @@
+    if '-BAND' in wl_band1:
+        wl_band1 = wl_band1[0]
+
+    return wl_band1
+
+def populate_obs_occultation_VGPPS_source_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    src_name1 = index_row['SIGNAL_SOURCE_NAME_1']
+    src_name2 = index_row['SIGNAL_SOURCE_NAME_2']
+
+    if src_name2 != 'N/A':
+        assert src_name1 == src_name2, 'Mismatched src_name1 and src_name2.'
+
+    return src_name1
+
+def populate_obs_occultation_VGPPS_host_PROF(**kwargs):
+    return 'voyager'
+
+
+### OBS_RING_GEOMETRY TABLE ###
+
+def populate_obs_ring_geometry_VGPPS_ring_radius1_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    radius1 = import_util.safe_column(index_row, 'MINIMUM_RING_RADIUS')
+
+    return radius1
+
+def populate_obs_ring_geometry_VGPPS_ring_radius2_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    radius2 = import_util.safe_column(index_row, 'MAXIMUM_RING_RADIUS')
+
+    return radius2
+
+def populate_obs_ring_geometry_VGPPS_resolution1_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    res = import_util.safe_column(index_row, 'MINIMUM_RADIAL_RESOLUTION')
+
+    return res
+
+def populate_obs_ring_geometry_VGPPS_resolution2_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    res = import_util.safe_column(index_row, 'MAXIMUM_RADIAL_RESOLUTION')
+
+    return res
+
+def populate_obs_ring_geometry_VGPPS_proj_resolution1_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    res = import_util.safe_column(index_row, 'MINIMUM_RADIAL_RESOLUTION')
+
+    return res
+
+def populate_obs_ring_geometry_VGPPS_proj_resolution2_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    res = import_util.safe_column(index_row, 'MAXIMUM_RADIAL_RESOLUTION')
+
+    return res
+
+def populate_obs_ring_geometry_VGPPS_j2000_longitude1_PROF(**kwargs):
+    return None
+
+def populate_obs_ring_geometry_VGPPS_j2000_longitude2_PROF(**kwargs):
+    return None
+
+def populate_obs_ring_geometry_VGPPS_ring_azimuth_wrt_observer1_PROF(**kwargs):
+    return None
+
+def populate_obs_ring_geometry_VGPPS_ring_azimuth_wrt_observer2_PROF(**kwargs):
+    return None
+
+def populate_obs_ring_geometry_VGPPS_phase1_PROF(**kwargs):
+    return 180.
+
+def populate_obs_ring_geometry_VGPPS_phase2_PROF(**kwargs):
+    return 180.
+
+# Source is star, observer is Voyager
+# Source DELTA SCO is at south, observer Voyager is at north. Target: S ring.
+# Source SIGMA SGR is at south, observer Voyager is at north. Target: U ring.
+# Source SIGMA SGR is at north, observer Voyager is at south. Target: N ring.
+# Source BETA PER is at north, observer Voyager is at south. Target: U ring.
+
+# Incidence angle: the angle between the point where incoming source photons
```

Comment:
Fix comments to agree with updated comments from VGISS PROF throughout file

Reply:
done